### PR TITLE
Bug #13388

### PR DESCRIPTION
**Pull request by mmoqui**
Fix bug.
The issue was comming from the check of specific rights wasn't good. To cover all the cases, it is required once the topic is created to check it has only local rights and hence not inherited ones from a parent topic. Then, we have to recursively to fetch only the admin right profile from a parent (the not inherited one or the one of the component instance itself).

### CODE REVIEW

File: kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java [hunk 1]
```diff
@@ -758,15 +758,16 @@ public synchronized NodePK addSubTopic(NodeDetail nd, String alertType, String p
...
+    // to prevent no access to users having some rights with the father topic
+    NodeDetail created = getNodeHeader(pk.getId());
+    if (created.haveLocalRights()) {
       ProfileInst parentProfile;
       String profileAdmin = SilverpeasRole.ADMIN.getName();
       if (NodePK.ROOT_NODE_ID.equals(parentId)) {
         parentProfile = getProfile(profileAdmin);
       } else {
-        parentProfile = getTopicProfile(profileAdmin, parentId);
+        parentProfile = getRecursiveTopicProfile(profileAdmin, parentId);
```
Comment (SilverYoCha) [inline kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java:770]: I don't understand why it is necessary to get the parent administrator profiles recursively.
If a user creates a folder, it means that he has an administrator profile on the folder he creates from. And therefore, the parent folder of the created folder necessarily has the administrator profile.
Am I missing something?

**Review by SilverYoCha — CHANGES_REQUESTED** [review]

File: kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java (hunk 1 above)
Comment (mmoqui) [reply kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java:770]: A folder with specific rights can be created into a path of folders with, among them, a folder with specific rights. The goal here is to fetch all the administrative rights of this folder to set them as default administrator to the new folder. And we have to take into account this new folder is in a path of folders in which no one has specific rights: the administrators to copy for the new folder should be then the one of the root folder, id est the Kmelia instance.

Comment (SilverYoCha) [reply kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java:770]: How do you get the situation where the new folder is created into a path where no one has specific rights?

Comment (SilverYoCha) [reply kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java:770]: OK, Miguel gave me a phone call to explain to me the different cases.

**Comment by SilverYoCha** [comment]
Merged.
Closed manually because of manual git operations performed during integration.

**Pull request closed without merging**